server/models/Profile.js

```diff
@@ -0,0 +1,132 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  user_username: {
```

Comment:
Our `User` model already contains the `username` field, we wouldn't want to duplicate it :)

Reply:
I was attempting to use the [Subset Pattern](https://docs.mongodb.com/manual/tutorial/model-embedded-one-to-one-relationships-between-documents/) that I found on the MongoDB website here. Is it a good idea or should I just remove `user_username`?

Reply:
This would actually just be storing the username on the profile. This means we would need to query the database specifically by the username. That's not something I would like to do, `userId` would be better to reference here. 

Subset pattern is great for breaking up large collections which ultimately can be a performance benefit. User collection is not very large so we don't need to really worry about sub dividing our collections out. 

👍🏻